libcodechecker/util.py

```diff
@@ -549,4 +549,8 @@ def get_last_mod_time(file_path):
     """
     Return the last modification time of a file.
     """
-    return os.stat(file_path)[9]
+    try:
+        return os.stat(file_path)[9]
```

Comment:
What is this magic number `9`? The return value of os.stat is an object whose members are `st_mtime`, `st_ctime` etc.:
https://docs.python.org/2/library/os.html